<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AWS Script</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="local">
     <item>
      <widget class="QPushButton" name="bNYTask">
       <property name="text">
        <string>NY Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bTest">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bDraw">
       <property name="text">
        <string>Draw</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="all">
     <item>
      <widget class="QPushButton" name="bDrawAll">
       <property name="text">
        <string>Draw All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
